syserol/model.py

```diff
@@ -106,40 +107,42 @@ def test_predictions(function="ADCD"):
     return corr
 
 
+@lru_cache()
 def cross_validation():
     """ 10 Fold Cross Validation to Test Predictive Abilities"""
     cube, glyCube = createCube()
-    _, mapped = importFunction()
+    _, mapped = importFunction() 
     glycan, _ = importGlycan()
 
     X = glyCube
+    copy = glyCube #copy glyCube
     index = []
     original = []
```

Comment:
It's much better to make the array at the beginning, then fill in elements through indexing. This helps to ensure that you are doing what you expect, because if the indexes don't line up you'll get an error. For example: `original = np.zeros(10)`